models/organization/src/team/transformers.ts

```diff
@@ -0,0 +1,16 @@
+import { Transformer } from '@commercetools-test-data/core';
+import type { TTeam, TTeamGraphql } from './types';
+
+const transformers = {
```

Comment:
Maybe what @emmenko meant was to return slightly different objects from here and keep just one Team model.

Reply:
Yes correct. I think the Team model can use list of references by default but then in the graphql representation we need to change it to list of users.

I think we can use the `replaceFields` function for that. See maybe other models if they have a similar example.

Reply:
Thanks for the clarification and guidance! 🙌 I've implemented those suggestions.
Now, the default behavior is to create the REST model. For the GraphQL version, the `members` field is replaced, and `membersRef` and `__typename` fields are added in the GraphQL transformer.
Covered in 918296166d910994106e9a746876c0a81ca59f64